modules/standard/LinkedLists.chpl

```diff
@@ -203,6 +203,52 @@ record LinkedList {
      return ret;
    }
 
+  /*
+    find `e` in the list.
+
+    :arg e: The element to be checked for
+    :return: True if the item was found, otherwise returns false
+    :rtyle: `bool`
+   */
+  proc contains(e : eltType): bool {
+    for item in this {
+      if (e == item) {
+        return true;
+      }
+    }
+    return false;
+  }
+  
+  /*
+    Returns the data stored in the first element of the list
+    
+    :return: the data stored at the front of the list
+    :rtype: `ref eltType`
+    :throws SystemError: if the list is empty
+   */
+  proc ref front() ref: eltType throws {
```

Comment:
Stylistically, we've chosen to make other containers such as `list` halt instead of throw. So for now, you can go ahead and remove the `throws` tag.

While it's true that the `!` operator does throw, it should be safe to just go ahead and `try!` since we know that it realistically can never halt (if our linked list isn't empty, how can `first!` result in `nil`?).

---

You should also add a warning explaining that this method will halt if the linked list is empty. For reference ,see the documentation of `list.first`.